tests/test_relation_verification.py

```
@@ -53,7 +53,7 @@ def do_GET(self) -> None:  # noqa: N802 -- BaseHTTPRequestHandler API
         self.end_headers()
         self.wfile.write(body)
 
-    def log_message(self, format: str, *args) -> None:  # noqa: A002 -- stdlib signature
+    def log_message(self, format: str, *args) -> None:
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)